db/schema.rb

```diff
@@ -11,11 +11,22 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20160912232500) do
+ActiveRecord::Schema.define(version: 20160913145325) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "categories", force: :cascade do |t|
+    t.string   "name"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+  end
+
+  create_table "categorizations", id: false, force: :cascade do |t|
```

Comment:
Can we come up with a better name for the join table? Maybe "collections" or "product_collections" or something? Categorizations seems a bit vague to me.